packages/mobile/src/components/add-funds-drawer/AddFundsDrawer.tsx

```diff
@@ -70,7 +70,10 @@ export const AddFundsDrawer = () => {
     if (selectedPurchaseMethod === PurchaseMethod.CARD) {
       openCardFlow()
     } else if (selectedPurchaseMethod === PurchaseMethod.CRYPTO) {
-      openUSDCManualTransferModal()
+      openUSDCManualTransferModal({
+        isOpen: true,
```

Comment:
I don't think `isOpen: true` should be needed?

Reply:
hm i get type errors when i omit it

Reply:
thought so too tho looking at `createModal` but idk

Reply:
figured it out - need to omit `isOpen` on the type of the modal